native-extension/klarna/checkout/klarna-checkout/projects/klarna-checkout/src/lib/klarna-checkout/klarna-checkout.component.spec.ts

```diff
@@ -0,0 +1,25 @@
+import { ComponentFixture, TestBed } from '@angular/core/testing';
+
+import { KlarnaCheckoutComponent } from './klarna-checkout.component';
+
+describe('KlarnaCheckoutComponent', () => {
+  let component: KlarnaCheckoutComponent;
+  let fixture: ComponentFixture<KlarnaCheckoutComponent>;
+
+  beforeEach(async () => {
+    await TestBed.configureTestingModule({
+      declarations: [ KlarnaCheckoutComponent ]
+    })
+    .compileComponents();
+  });
+
+  beforeEach(() => {
+    fixture = TestBed.createComponent(KlarnaCheckoutComponent);
+    component = fixture.componentInstance;
+    fixture.detectChanges();
+  });
+
+  it('should create', () => {
+    expect(component).toBeTruthy();
```

Comment:
I think we should unit test this. Isn't that part of SAP's guidelines for publishing things?

Reply:
> Why is the directory structure /klarna/checkout/klarna-checkout? Will there be future files in /klarna and /klarna/checkout?

I went with Richard's suggestion

Reply:
> I think we should unit test this. Isn't that part of SAP's guidelines for publishing things?

Maybe. We would probably need to setup some kind of pipeline.

Reply:
What was the reasoning for Richard's suggestion?

Reply:
I don't know, frankly I don't think it matters. This is not the permanent location for this

Reply:
Frankly I do think it matters, unless you create a ticket to move it to its correct place. This is for public consumption.